packages/docs-app/src/examples/core-examples/breadcrumbsExamples.tsx

```diff
@@ -11,12 +11,12 @@ import { CodeExample, type ExampleProps } from "@blueprintjs/docs-theme";
 export const BreadcrumbsBasicExample: React.FC<ExampleProps> = props => {
     const code = dedent`
         <Breadcrumbs
-        items={[
-            { text: "Blueprint" },
-            { text: "Docs" },
-            { text: "Components" },
-            { text: "Breadcrumbs" },
-        ]}
+            items={[
+                { text: "Blueprint" },
+                { text: "Docs" },
+                { text: "Components" },
+                { text: "Breadcrumbs" },
+            ]}
```

Comment:
Fixes a minor indentation typo in the breadcrumbs docs: [before](https://blueprintjs.com/docs/#core/components/breadcrumbs.basic-breadcrumbs) / [after](https://output.circle-artifacts.com/output/job/39b0ad4f-4874-4167-a82c-396ac571a55a/artifacts/0/packages/docs-app/dist/index.html#core/components/breadcrumbs.basic-breadcrumbs)